server/routes/hearingOutcome/damagesOwed/damagesOwedValidation.ts

```diff
@@ -0,0 +1,38 @@
+import { FormError } from '../../../@types/template'
+
+type DamagesOwedForm = {
+  damagesOwed: string
+  amount: string
+}
+
+const errors: { [key: string]: FormError } = {
+  MISSING_DAMAGES_OWED_CHOICE: {
+    href: '#damagesOwed',
+    text: 'Select yes, if any money is being recovered for damages.',
+  },
+  MISSING_AMOUNT: {
+    href: '#amount',
+    text: 'Enter amount being recovered.',
+  },
+  NAN: {
+    href: '#amount',
+    text: 'Numerical values only.',
```

Comment:
I think we need to get Anne to write us a message for this one as I can't find anything for it on the spreadsheet. I'll pop her a message but we can change it next week once this is merged